src/client/include/com/amazonaws/kinesis/video/client/Include.h

```diff
@@ -928,6 +940,15 @@ typedef enum {
     CONTENT_STORE_PRESSURE_POLICY_DROP_TAIL_ITEM = 1,
 } CONTENT_STORE_PRESSURE_POLICY;
 
+typedef enum {
+    // base case
+    STREAM_EVENT_TYPE_NONE = 0,
+    // To trigger image generation and persistence in s3
+    STREAM_EVENT_TYPE_IMAGE_GENERATION,
+    // To receive a notification that fragment has been persisted

```

Comment:
can we qualify to be fragment ingestion status on persistence e.g.
STREAM_EVENT_TYPE_PERSISTED_NOTIFICATION
will help us extend later with STREAM_EVENT_TYPE_RECEIVED_NOTIFICATION for real-time getmedia path.

Reply:
as we discussed offline, this can be revisited for other fragment ingestion status notifications later.